lib/index.js

```diff
@@ -11,23 +11,28 @@ module.exports = {
    * @param {Function} callback - callback function with arguments (error, object)
    */
   getOptions (language, variant, callback) {
-    var validCodegen = languageMap.filter((codegen) => {
-      var lang = codegen.lang.trim(),
-        currentVariant = codegen.variant.trim();
-      return language === lang.toLowerCase() && variant.toLowerCase() === currentVariant.toLowerCase();
-    });
+    try {
+      var validCodegen = languageMap.filter((codegen) => {
+        var lang = codegen.lang.trim(),
+          currentVariant = codegen.variant.trim();
+        return language === lang.toLowerCase() && variant.toLowerCase() === currentVariant.toLowerCase();
+      });
 
```

Comment:
lib/index.js

Reply:
thank you team for your point of directions

Reply:
Is there a review here? I didn't get you